src/main/java/producer/FluxProducer.java

```diff
@@ -0,0 +1,29 @@
+package producer;
+
+import java.util.Map;
+import java.util.Properties;
+
+public class FluxProducer<K, V> implements Producer {
+    private Map<String, Object> configs;
```

Comment:
What is the `configs` var?

Reply:
see the example and constructors here https://kafka.apache.org/27/javadoc/org/apache/kafka/clients/producer/KafkaProducer.html

Reply:
Got it